pulp_maven/app/maven_deploy_api.py

```diff
@@ -77,9 +77,12 @@ class MavenApiViewSet(APIView):
     authentication_classes = []
     permission_classes = []
 
-    def redirect_to_content_app(self, distribution, relative_path):
+    def redirect_to_content_app(self, distribution, relative_path, request):
```

Comment:
I think this is good for now. I'm wondering if we should be using the django machinery to build this URL instead. Would you be willing to look for 3-5 minutes at the django "building url" code and see if we're duplicating stuff with this?